scripts/localchat.lic

```diff
@@ -16,9 +16,15 @@
     - bugfix for default dialog text
   v1.0.0 (2023-07-01)
     - initial release
+  v1.0.1 (2023-07-04)
+    - chore: refactor loop for simplicity and to be more rubyish
+    - bugfix: removed unintentional delay in closing the window with the close button
+    - bugfix: no longer displays non-dialog lines through better synchronization
```

Comment:
This should really be 1.0.2 and update the version number above it. Otherwise, all looks good!

Reply:
Right you are, thank you!